pumpkin-lib/src/constraints/cumulative.rs

```diff
@@ -112,7 +115,6 @@ pub fn cumulative<Var: IntegerVariable + 'static + Debug>(
     durations: &[i32],
     resource_requirements: &[i32],
     resource_capacity: i32,
-    allow_holes_in_domain: bool,
 ) -> impl Constraint {
```

Comment:
Reduce the documentation duplication. We can reference the `cumulative_with_options` as the main documentation from this `cumulative` function. Explicitly mention `CumulativeOptions::default()` is given to `cumulative_with_options`.